src/js/elements/mapchip/add_filter_button.js

```diff
@@ -0,0 +1,28 @@
+import { Observable } from "../../utils";
+
+export class AddFilterButton extends Observable {
```

Comment:
you should be extending Component, not observable

Reply:
done